src/components/Select/index.tsx

```diff
@@ -25,6 +34,7 @@ export const Select = ({ isMulti = false, onChange, dataQa = null, dataQaProps =
         noOptionsMessage={noOptionsMessage}
         components={{
           DropdownIndicator,
+          Option: !props.formatOptionLabel ? Option : components.Option,
```

Comment:
if the component use `formatOptionLabel`, then `formatOptionLabel` should be the one responsible for displaying the tooltip

Reply:
Does this component included in the docs? or if it is not yet, you can put this in to a comment

Reply:
what do you mean? sorry, i don't catch it